content/outcomes/application-development/prioritizing-outcomes.md

```diff
@@ -0,0 +1,170 @@
+
+GIVEN `situation or context`
+
+WHEN `some action is carried out`
+
+THEN `some consequences`
+
+
+{{% callout %}}
+Example:
+
+**GIVEN** _I visit the proposal summary page_
+
+**WHEN** _I click the button labeled “PDF Download”_
+
+**THEN** _a PDF file of the sales proposal downloads to my computer_
+{{% /callout %}}
+
+Acceptance criteria are important because they **clearly describe when the story is _done_**. Without an explicit statement of “done”, the feature risks [scope creep](https://en.wikipedia.org/wiki/Scope_creep). In the above example, the feature is done when the sales proposal  downloads to the user's computer as a PDF file. 
+
+Would it be handy if the user could email the PDF directly from the product? Sure!
+
+Would it be great if the system would automatically prompt the user to follow up on the sale proposal in one week? Heck yeah!
+
+_But_, those features are _out of scope for this story_. The team should create new stories for those features with their own very explicit and tightly focused acceptance criteria. 
+
+#### Resources
+
+Attach anything else helpful explaining the story including mockups, sketches or user flows.
+
+## Describe the advantages of estimating work in terms of risk and complexity instead of time
+
+Simply put: [people are really bad at estimating based on time](https://en.wikipedia.org/wiki/Planning_fallacy). Yet, most projects, software or otherwise, insist on doing so. We recommend estimating tasks based on risk and complexity rather than time to completion.
+
+Agile software teams use **estimation** to understand the relative complexity of delivering a story. They do this because:
+
+* It helps the team understand and compare the development cost of new features, and make decisions accordingly.
+* It allows the team to get to a shared understanding of the story and the risks or unknowns they could encounter.
+* Over time, estimates can be used to predict how long it will take to deliver a particular feature or release.
+
+When discussing a story, usually in an [iteration or sprint planning meeting](https://tanzu.vmware.com/developer/practices/ipm/), teams will estimate the size of a story in terms of **story points**. Story points measure a story’s complexity and risk. Teams use different scales. For example:
+ 
+- A simple scale of 1, 2, or 3 points 
+- "T-shirt sizing" as Small (S), Medium (M) Large (L), Extra Large (XL)
+- The Fibonacci sequence: 1, 2, 3, 5, 8, 13 (or more) points
+
+We use a scale rather than time estimates because time estimates are known to be highly error prone. They do not account well for the uncertainty that teams often encounter when building software. Time estimates are especially prone to [planning fallacy](https://en.wikipedia.org/wiki/Planning_fallacy) and [optimism bias](https://en.wikipedia.org/wiki/Optimism_bias).
```

Comment:
First sentence: Remove "a" between "use" and "scale".

Reply:
Done